.github/workflows/release.yml

```diff
@@ -7,7 +7,7 @@ env:
   GITHUB_TOKEN: ${{ secrets.PULUMI_BOT_TOKEN }}
 
 jobs:
-  release-pulumi-language-dotnet:
+  release-pulumi-language-yaml:
```

Comment:
Hah, actually a useful change here :)

Reply:
Yeh copy paste fun, noticed it when the job ran